improve explanation on Java array .length property (#2855)
exercism/java: concepts/arrays/about.md

Accessing an index that is outside of the valid indexes for the array results in an `IndexOutOfBoundsException`.

Arrays can be manipulated by either calling an array instance's methods or properties, or by using the static methods defined in the `Arrays` class (typically only used in generic code).

> Java has a built-in `length` property that returns the length of an array, which can be accessed like this:

Reviewer: I think the wording sounds like the property is on Java itself. I'd also suggest removing the word "returns" since it is property rather than a method that returns something. For example:

```suggestion
The `length` property holds the length of an array.
It can be accessed like this:
```
